Makefile

```diff
@@ -82,6 +82,19 @@ lint: ## run linter(s)
 	--build-arg GIT_TAG=$(GIT_TAG) \
 	--target lint
 
+.PHONY: docs
+docs: ## generate documentation
+	@docker build . \
+	--output type=local,dest=./docs/ \
```

Comment:
`output` will not rsync so if some files are not their anymore they will still be in working tree. I suggest to do smth like https://github.com/docker/cli/blob/11c76f1ddbb64e4d88b44b48c017b81ffe86e0e5/docker.Makefile#L84-L88

Reply:
done, thanks @crazy-max 🙏